tests/python-pytest/onnx/export/backend_rep.py

```diff
@@ -72,13 +72,22 @@ def run(self, inputs, **kwargs):
         for idx, input_name in enumerate(data_names):
             data_shapes.append((input_name, inputs[idx].shape))
 
-        mod = mx.mod.Module(symbol=self.symbol, data_names=data_names, context=ctx,
-                            label_names=None)
-        mod.bind(for_training=False, data_shapes=data_shapes,
-                 label_shapes=None)
-        mod.set_params(arg_params=self.arg_params, aux_params=self.aux_params)
+        # module bind method requires all data to have same batch size,
```

Comment:
duplicate in onnx_export.py

Reply:
This is in the test for export. the changes in onnx_export.py are used with create_onnx_graph_proto()